CheckYourEligibility-Parent/Views/Check/Enter_Child_Details.cshtml

```diff
@@ -42,8 +46,14 @@
                 <p class="govuk-error-message">
                     <span asp-validation-for="ChildList[i].FirstName"></span>
                 </p>
-
-                <input asp-for="ChildList[i].FirstName" id="ChildList[@i].FirstName" class="govuk-input govuk-!-width-two-thirds" type="text" aria-label="First name">
+                   
+                    @if (childIndex ==i)
+                    {
+                        <input asp-for="ChildList[i].FirstName" id="ChildList[@i].FirstName" class="govuk-input govuk-!-width-two-thirds" type="text" aria-label="First name" autofocus/>
+                    } else 
+                    {
+                       <input asp-for="ChildList[i].FirstName" id="ChildList[@i].FirstName" class="govuk-input govuk-!-width-two-thirds" type="text" aria-label="First name"/>
```

Comment:
I would choose a ternary statement inside the `<input` instead of two if clauses, but it won't matter.

Reply:
I don't think I'm able to put a ternary expression directly into an attribute because of the conditional element rendering. Did you have something specific in mind here?